addon/components/plugins/text-style/color.ts

```diff
@@ -0,0 +1,98 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { SayController } from '@lblod/ember-rdfa-editor';
+
+import IntlService from 'ember-intl/services/intl';
+import { TextSelection } from 'prosemirror-state';
+
+type Args = {
+  controller?: SayController;
+};
+export default class Color extends Component<Args> {
+  @service declare intl: IntlService;
+  colors = ['black', 'red', 'green', 'blue', 'orange'];
+
+  get controller() {
+    return this.args.controller;
+  }
+
+  get currentStyle() {
+    const currentColor = this.colors.find(this.colorIsActive);
+    if (currentColor) {
+      return `${this.intl.t('ember-rdfa-editor.color')} ${currentColor}`;
+    } else {
+      return this.intl.t('ember-rdfa-editor.default-color');
+    }
+  }
+
+  @action
+  enableColor(color: string) {
+    if (this.controller) {
+      const state = this.controller.mainEditorState;
+      const { selection } = state;
+      const { $from, $to, $cursor } = selection as TextSelection;
+      const markType = state.schema.marks.color;
+
+      const mark = markType.create({ color });
+      if (markType && $cursor) {
+        if (markType.isInSet(state.storedMarks || $cursor.marks())) {
+          this.controller.withTransaction((tr) => {
+            tr.removeStoredMark(markType);
+            return tr.addStoredMark(markType.create({ color }));
```

Comment:
you defined this as a constant on line 37, why not use it? (same thing on line 46)